libs/wire-api/src/Wire/API/User/Search.hs

```diff
@@ -370,3 +375,9 @@ instance S.ToParamSchema EmailVerificationFilter where
     mempty
       & S.type_ ?~ S.OpenApiString
       & S.enum_ ?~ ["unverified", "verified"]
+
+data SetSearchable = SetSearchable
+  { setSearchable :: Bool
+  }
+  deriving (Generic)
+  deriving anyclass (Aeson.ToJSON, Aeson.FromJSON, S.ToSchema)
```

Comment:
This would create camelCase JSON fields. We use snake_case everywhere. Can you please use `ToSchema` from `Data.Schema` and then deriving `ToJSON`, `FromJSON` and `S.ToSchema`.